### ISSUE TITLE
[bug] Problem with ``parse_musl_libc``

### ISSUE THREAD

**Issue by memsharded** [conan-io/conan#15833]
### Environment details

Introduced in https://github.com/conan-io/conan/pull/15683

cc @jwillikers 

```python
 if ldd_output is None:
     first_line = ldd_output.partition("\n")[0]
```

This is broken and will crash. Realized because the IDE linter was marking it.

Could you please have a look, retry locally and check what is happening, and how could it be working for your case in the past?

Many thanks!

**Comment by jwillikers** [comment]
Yes, I can take a look. I'd guess that I hit undo or did something similarly clumsy in my editor.

**Comment by jwillikers** [comment]
Just a run-of-the-mill typo. Sorry 'bout that. Fixed in #15834.

**Comment by memsharded** [comment]
Thanks very much!
